flopy/modpath/mp6sim.py

```diff
@@ -9,6 +9,14 @@
 """
 import numpy as np
 
+pandas_available = False
```

Comment:
pandas is considered an optional dependency, and we have recently changed the import format for these.  There are examples throughout the flopy code.  

```
        pd = import_optional_dependency(
            "pandas",
            error_message="get_dataframe() requires pandas.",)
```

Reply:
I have implement this (on next commit).  Are these standard practices documented somewhere I missed (entirely possible); if not it would be helpful if they were detailed in /CONTRIBUTING.md